pkg/scheduler/factory.go

```diff
@@ -476,8 +476,8 @@ func NewPodInformer(client clientset.Interface, resyncPeriod time.Duration) core
 }
 
 // MakeDefaultErrorFunc construct a function to handle pod scheduler error
-func MakeDefaultErrorFunc(client clientset.Interface, podLister corelisters.PodLister, podQueue internalqueue.SchedulingQueue, schedulerCache internalcache.Cache) func(*framework.QueuedPodInfo, error) {
-	return func(podInfo *framework.QueuedPodInfo, err error) {
+func MakeDefaultErrorFunc(client clientset.Interface, podLister corelisters.PodLister, podQueue internalqueue.SchedulingQueue, schedulerCache internalcache.Cache) func(*framework.QueuedPodInfo, error, string) {
+	return func(podInfo *framework.QueuedPodInfo, err error, nnn string) {
```

Comment:
As a followup, we should just have this as a function in scheduler.go, we don't need to ```MakeDefaultErrorFunc```

Reply:
sg